### DISCUSSION TITLE
Multiple instances of mcmanager

### DISCUSSION THREAD

**Discussion by JonEBee** [nextgenhealthcare/connect#6298] [Q&A] [answered]
Mirth 4.4.1 running on a VM with Windows Server 2019 Standard version 1809, 11.7 GB RAM and 266 GB of 500GB disk space used.

When looking at the Process List in the Task Manager, I see multiple instances of mcmanager started and none of them are doing anything.  Is this normal and, if not, how do I ensure that only one is running at any one time?  Leaving only one instance running will open up about 90MB per excess instance of memory for other uses.

**Comment by pacmano1** [comment]
Are multiple RDP sessions open on the server?

**Comment by JonEBee** [comment]
As best as I can tell, there is only one user allowed on the Windows server at any time.  When I am leaving an active session, I use the Disconnect option to close the RDP connection.  There have been occasions where my internet service collapsed and the RDP session was cut off, though, which would leave sessions open.  We use the Mirth administrator launcher to create Mirth client access.

**Reply by pacmano1** [reply, marked as answer]
"there is only one user allowed on the Windows server at any time" So RDP is not enabled?  Or you have group or local policy enforcing that?   Of just a management directive?   

mcmanager,exe is that system task tray item that starts on login.  You don't need to install it at all.   You can start/stop mirth with windows services manager.  You can edit mirth.properties to set things it can set.    
You can remove it from startup.
